src/redux/wallets/selectors.ts

```diff
@@ -48,8 +48,10 @@ export const getAccountTransactions = (
     );
     const transactions = getSelectedWallet(state).transactions;
     if (transactions) {
-        return Object.values(getSelectedWallet(state).transactions).filter(
-            tx => tx.address === account.address
+        return Object.values(transactions).filter(
+            tx =>
+                tx.address.toLowerCase() === account.address.toLowerCase() ||
+                tx.toAddress.toLowerCase() === account.address.toLowerCase()
```

Comment:
blockchain verification should be added here to make sure the address is not on different blockchains